source/adapters/level_zero/command_buffer.cpp

```diff
@@ -719,6 +766,113 @@ urCommandBufferFinalizeExp(ur_exp_command_buffer_handle_t CommandBuffer) {
   return UR_RESULT_SUCCESS;
 }
 
+namespace {
```

Comment:
Slight preference for having a single anonymouse namepace at the top of the file rather than several interspersed throughout.

Reply:
I structured the code so that functions that are relevant to multiple entrypoints are at the top of the file. While functions that are only used by a single entrypoint would be right above that entrypoint.

I have a slight preference for this structure but am not opposed to changing if we think that having everything at the top is better. @EwanC do you have a preference?

Reply:
As discussed offline, will keep the structure with the multiple unnamed namespaces.